src/templates/default/template.jsx

```diff
@@ -1,90 +1,92 @@
 var template = `
-<div class=" mx-auto bg-white p-7 m-5">
-  <div class="flex row justify-between">
-    <div>
-      {{{company_name}}} <br/>
-      {{{your_name}}} <br/>
-      {{{company_gstin}}} <br/>
-      {{{address}}} <br/>
-      {{{city}}} <br/>
-    </div>
-    <div>
-      {{{company_logo}}}
-    </div>
-  </div>
-  <div class="flex row justify-between mt-10">
-    <div class="w-2/4">
-      {{{bill_to}}} <br/>
-      {{{client_company}}} <br/>
-      {{{client_name}}} <br/>
-      {{{client_gstin}}} <br/>
-      {{{client_address}}} <br/>
-      {{{client_city}}} <br/>
-    </div>
-    <div class="w-2/4">
-      <table class="w-full table-fixed" ><tr><td>{{{invoice_number_label}}}</td></div><td>{{{invoice_number}}}</td></tr><tr><td>{{{invoice_date_label}}}</td><td>{{{invoice_date}}}</td></tr><tr><td>{{{due_date_label}}}</td><td>{{{due_date}}}</td></tr></table>
-    </div>
```

Comment:
Why is this template changed ?

Reply:
I've added id="template" in both the templates inorder to capture the child components for printing from client side